R/survey_statistics.r

```diff
@@ -420,20 +409,20 @@ survey_median <- function(
   if (!is.null(vartype)) {
     vartype <- if (missing(vartype)) "se" else match.arg(vartype, several.ok = TRUE)
   }
-  if (missing(interval_type) & !inherits(.svy, "svyrep.design")) interval_type <- "Wald"
-  if (missing(interval_type) & inherits(.svy, "svyrep.design")) interval_type <- "probability"
+  if (missing(interval_type) & !inherits(.svy, "svyrep.design")) interval_type <- "mean"
```

Comment:
Single line again here

```
if (missing(interval_type)) interval_type <- "mean"
```

Reply:
Should also be resolved by https://github.com/gergness/srvyr/pull/124/commits/f1a4cd92f8c8c03e9766495b5e0299ef6fc77d7b.